### ISSUE TITLE
Deterministic builds

### ISSUE THREAD

**Issue by public** [pyca/cryptography#430]
Tor and BitCoin use Gitian for this.  http://gitian.org/

**Comment by alex** [comment]
This is fundamentally blocked by deterministic build support in distutils, setuptools, cffi, etc.

I don't think we have the bandwidth to make all those happen, and it's not a priority ATM, so closing this. We can reopen if there's movement in this space.
